jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/ConnectorFactory.java

```diff
@@ -106,6 +108,13 @@ private SslConnectionFactory newSslConnectionFactory() {
             factory.setSecureRandomAlgorithm(sslConfig.prng());
         }
 
+        // NOTE: ^TLS_RSA_.*$ ciphers are disabled by default in Jetty 9.4.12+ (https://github.com/eclipse/jetty.project/issues/2807)
+        // JDisc will allow these ciphers by default to support older clients (e.g. Java 8u60 and curl 7.29.0)
+        String[] excludedCiphersWithoutTlsRsaExclusion = Arrays.stream(factory.getExcludeCipherSuites())
+                .filter(cipher -> !cipher.equals("^TLS_RSA_.*$"))
+                .toArray(String[]::new);
+        factory.setExcludeProtocols(excludedCiphersWithoutTlsRsaExclusion);
```

Comment:
Why is setExcludeProtocols() used and not setExcludeCipherSuites()? Am I missing something?

Reply:
Hmm, good point. Looks like it should be cipher suites: https://github.com/eclipse/jetty.project/pull/2855/files

Reply:
It's a typo, it should have been `setExcludeCipherSuites()`